Snabble/UI/Utilities/DynamicFont.swift

```diff
@@ -6,6 +6,8 @@
 
```

Comment:
I'd would suggest to remove the extension on `UILabel` the method does to many things.

I don't expect the method to change `numberOfLines` and `lineBreakMode`. `adjustFontForContentSizeCategory` can be talked about, but I'd like to have it separated. It's what each developer knows.

Actually I'd say we only need the extension for `UIButton`. That's just a nice handy extension to set the preferredTextStyle. I would also call it `.preferredFont(forTextStyle:, weight:)` to have the same API as UIKit.